doc/classes/Tween.xml

```diff
@@ -510,7 +516,7 @@
 		<signal name="step_finished">
 			<param index="0" name="idx" type="int" />
 			<description>
-				Emitted when one step of the [Tween] is complete, providing the step index. One step is either a single [Tweener] or a group of [Tweener]s running in parallel.
+				Emitted when one step of the [Tween] is complete, providing the step index. One step is either a single [Tweener] or a group of [Tweener]s running in parallel. Use the method [method get_step_count] to determine how many steps are in the [Tween] in total.
```

Comment:
I'm not sure if this extra note is particularly useful here.

Reply:
Hm, maybe not. My initial thought was that a common use case could be comparing the current step to the total number of steps, to determine when the tween was finished:
```gdscript
my_tween.step_finished.connect(_on_step_finished)
...
func _on_step_finished(step_idx: int):
    if step_idx == my_tween.get_step_count() - 1:
        print("Tween is finished!")
    else:
        print("The tween has only partially completed")
```

However, they could just use the `finished` signal instead. I suppose the note here more generally ties together the concept of "steps" across the Tween documentation?

Reply:
Same 😅 Pretty much all of the use cases I can imagine for it, I feel like would be better handled with specifically placed `CallbackTweener`s. Makes me wonder which of them came first...

Reply:
They were added together when the new Tweens were implemented.